src/components/sidebar/tabs/modelLibrary/ModelTreeLeaf.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div ref="container" class="model-lib-node-container">
+    <TreeExplorerTreeNode :node="node"> </TreeExplorerTreeNode>
+  </div>
+</template>
+
+<script setup lang="ts">
+import TreeExplorerTreeNode from '@/components/common/TreeExplorerTreeNode.vue'
+import { ComfyModelDef } from '@/stores/modelStore'
+import { RenderedTreeExplorerNode } from '@/types/treeExplorerTypes'
+
+const props = defineProps<{
+  node: RenderedTreeExplorerNode<ComfyModelDef>
+}>()
+</script>
+
+<style scoped>
+.model-lib-node-container {
```

Comment:
We can directly use tailwindcss in classes after https://github.com/Comfy-Org/ComfyUI_frontend/pull/768

Reply:
this was copypasted directly from NodeTreeLeaf

Reply:
Newly added code should directly use tailwindcss with class name.

Reply:
I think I applied what you mean now?